pages/notes.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title:  "Notes"
+permalink: /notes/
+---
+
+
+# For server implementors
+
+If you are implementing a CI server, please consider providing build information in the _cctray_ format, described on this site. This will make your server compatible with [CCMenu](http://ccmenu.org), [Nevergreen](https://github.com/build-canaries/nevergreen), and other tools that read the feed format.
+
+The specification leaves some open questions. If in doubt, please:
+* Provide times with a timezone, ideally in ISO8601 format with Zulu/GMT time marker.
```

Comment:
Because for example if we created a v2 of the spec I'd want this to be a **must** requirement.

Reply:
This however makes less sense to me in a generic notes section and I feel should be moved to the v1 spec.

It just doesn't make sense we wouldn't address these open questions if we made a v2 of the spec.

Reply:
To be honest, I'm doubtful that Jenkins will modify their weird behaviour around authentication, and I don't think we can/should make a statement about that in v2. We should standardise the time format. So, I wpould prefer to keep a general page with notes (aka. recommendations), but that shouldn't stop us from incorporating *some* recommendations into the next spec version.

Reply:
Makes sense